api_tests/lib_sdk/ledgers/config.ts

```diff
@@ -1,7 +1,7 @@
 import * as tmp from "tmp";
```

Comment:
I am not sure I agree with the `config.ts` file going under `ledgers`. For example it contains the `E2EActorConfig` I already mentioned but it also contains the `CndConfigFile` which is not related to a ledger.

I'd put the file in `lib_sdk/` for now and maybe do a 2nd PR where you split the config file (as it looks like something you'd like to do).

Reply:
good point, thanks